src/script/components/ParticipantItemContent/ParticipantItemContent.tsx

```diff
@@ -46,6 +46,7 @@ export interface ParticipantItemContentProps {
   hasUsernameInfo?: boolean;
   showArrow?: boolean;
   onDropdownClick?: (event: React.MouseEvent<HTMLButtonElement>) => void;
+  isUserContent?: boolean;
```

Comment:
I think we can drop this since now we know it's always going to be a `user` isn't it?